indico/modules/events/timetable/testing/fixtures.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.sessions.models.blocks import SessionBlock
+from indico.modules.events.timetable.models.breaks import Break
+from indico.modules.events.timetable.models.entries import TimetableEntry, TimetableEntryType
+
+
+@pytest.fixture
+def create_entry(db, dummy_event):
+    """Returns a a callable which lets you create timetable"""
+
+    def _create_entry(obj, start_dt):
+        data = {'start_dt': start_dt}
+        if isinstance(obj, Break):
```

Comment:
this whole block can go away - `object=obj` in the constructor is enough; no need to set the type manually either